pages/linux/lid.md

```diff
@@ -0,0 +1,9 @@
+# lid
+
+> Query ID database and report results.
+> On Fedora and Arch Linux, the binary name `lid` is taken by another program. See `tldr libuser-lid`.
+> More information: <https://example.org/>.
+
+- This page is currently a redirection stub:
```

Comment:
Maybe, a bit of clarification here to make it clear we're looking for contributors if we're merging this early?

```suggestion
- This page is currently a redirection stub (if you are familiar with this version of this command, please open a pull request!):
```